ci/travis/install.sh

```diff
@@ -21,3 +21,7 @@ conda update -q conda
 conda install pip conda-build anaconda-client
 # Useful for debugging any issues with conda
 conda info -a
+
+# Add channel to get boost libraries. Probably want to "self-host",
+# maybe embedding their recipes as a git module?
+conda config --add channels menpo
```

Comment:
That build failed. Once I got building working, installing failed because of the conda call. Luckily, it's easy to understand b/c the build script is not called when installing, which means that boost isn't there when trying to run diamond and things fall apart.

tl;dr Nothing can be installed inside build scripts by calling conda, they must be set as dependencies in the meta file.